src/SFA.DAS.Admin.Aan.Web.UnitTests/Validators/ManagerEvents/EventDateTimeViewModelValidatorTests.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using FluentAssertions;
 using FluentValidation.TestHelper;
 using SFA.DAS.Admin.Aan.Web.Models.NetworkEvent;
-using SFA.DAS.Admin.Aan.Web.Validators.ManageEvent;
+using SFA.DAS.Admin.Aan.Web.Validators;
 
-namespace SFA.DAS.Admin.Aan.Web.UnitTests.Validators;
+namespace SFA.DAS.Admin.Aan.Web.UnitTests.Validators.ManagerEvents;
```

Comment:
Looks like the folder name went wrong and the namespace as well. It should be ManageEvents not ManagerEvents

Reply:
I've fixed this in next branch already